.travis.yml

```diff
@@ -1,13 +1,14 @@
 language: python
 python:
-    - 3.4
+    - 3.5
     - 3.6
 install:
     - sudo apt-get -y update
     - sudo apt-get -y install python-matplotlib
-    - pip install codecov
+    - pip install -e .
+    - pip install -r test-requirements.txt
 script:
-    - nosetests
```

Comment:
@kamidzi You don't think nosetests are still worthwhile?

Reply:
I've replaced `nose` with `tox` and `pytest`. With travis, only `pytest` is executed. Locally, one could call `tox` (which also supports `nose`) to execute `pytest` for all the defined python versions, or simply execute `pytest` to test just the currently configured python version... I'll add a TESTING section to the README with this info.